src/main/resources/freemarker/data_custodian_approval.html

```diff
@@ -40,6 +40,7 @@
         </tr>
         <tr width="600" style="font-family: 'Roboto', sans-serif ;">
             <td align="left" style="font-family: 'Roboto', sans-serif ; font-size: 16px; color: #777777; text-align: left; line-height: 25px; padding: 0 30px 10px 30px;">
+                <!-- how do we want to update this reference to whitelist? -->
                 <p>To provide access to these individuals, you may add their email to the whitelist for the corresponding Terra workspace(s).</p>
                 <p>We recommend you provide access to these datasets via the Terra platform within the next 24 hours.</p>
                 <p>Access to data will not be provided by DUOS.</p>
```

Comment:
@solideoglori - We could use some editorial judgement for this email content

Reply:
Jonathan said we can just leave this email template as is because it is referencing the whitelist in terra workspace that still exists